daras_ai_v2/fal_ai.py

```
@@ -21,19 +23,30 @@ def generate_on_fal(model_id: str, payload: dict) -> typing.Generator[str, None,
-            yield "`" + logs[-1]["message"] + "`"
-        else:
-            repsonse_url = event_data["response_url"]
-            r = requests.get(repsonse_url, headers=_fal_auth_headers())
-            raise_for_status(r)
-            return r.json()
+    max_retries = 3
+    retry_delay = 1.0
+    for attempt in range(max_retries):
+        try:
+            for event_data in stream_sse_json(
+                requests.get(status_url, headers=_fal_auth_headers(), stream=True)
```

Ruff (0.15.1)

[error] Probable use of `requests` call without timeout

(S113)